osu.Game/Screens/Ranking/Expanded/Accuracy/AccuracyCircle.cs

```diff
@@ -215,12 +220,12 @@ private void load()
                     Padding = new MarginPadding { Vertical = -15, Horizontal = -20 },
                     Children = new[]
                     {
-                        new RankBadge(1, getRank(ScoreRank.X)),
-                        new RankBadge(0.95, getRank(ScoreRank.S)),
-                        new RankBadge(0.9, getRank(ScoreRank.A)),
-                        new RankBadge(0.8, getRank(ScoreRank.B)),
-                        new RankBadge(0.7, getRank(ScoreRank.C)),
-                        new RankBadge(0.35, getRank(ScoreRank.D)),
+                        new RankBadge(1, 1, getRank(ScoreRank.X)),
+                        new RankBadge(0.95, 0.95 + (0.025 - virtual_ss_percentage) / 2, getRank(ScoreRank.S)),
+                        new RankBadge(0.9, 0.9125, getRank(ScoreRank.A)),
+                        new RankBadge(0.8, 0.85, getRank(ScoreRank.B)),
+                        new RankBadge(0.7, 0.75, getRank(ScoreRank.C)),
```

Comment:
Do these new values have any mathematical significance, or is this all subjective? Why is this offset by 0.05 in 2 cases, 0.0125 in the third, and whatever else is that fourth expression? Does that last one have any numerical significance?

Reply:
I've changed this to use constants per your feedback. See the [relevant lines in the diff](https://github.com/ppy/osu/pull/22554/files#diff-6ad7f21c9f785ced34f1b5dea121ccf10e8ff29baae1b2ec9b08a30b4bae6cd3R235-R241).

Reply:
Much clearer, thanks.